### PR TITLE
golang format tools

### PR DESCRIPTION
**Pull request by mattmoor**
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party)`
/assign ImJasonH
/cc ImJasonH

### CODE REVIEW

File: pkg/entrypoint/entrypointer.go [hunk 1]
```diff
@@ -18,12 +18,13 @@ package entrypoint
...
 import (
 	"fmt"
-	"go.uber.org/zap"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"time"
 
+	"go.uber.org/zap"
+
```
Comment (sbwsg) [inline pkg/entrypoint/entrypointer.go]: odd spacing choice.  do we separate like this anywhere else in the codebase?

Comment (ImJasonH) [reply pkg/entrypoint/entrypointer.go]: Yeah the preferred style is to keep stdlib imports separate.

Comment (sbwsg) [reply pkg/entrypoint/entrypointer.go]: I meant the separation between zap and our internal packages.  We don't do that anywhere else.

Comment (ImJasonH) [reply pkg/entrypoint/entrypointer.go]: Oh yeah. Some people do that, some don't. I don't really care except I make it a policy never to disagree with formatting tools. 🤖

Comment (sbwsg) [reply pkg/entrypoint/entrypointer.go]: it's over.  the robots have won.

**Pull request merged**